Lombiq.Tests.UI/Extensions/HttpClientUITestContextExtensions.cs

```diff
@@ -129,16 +129,29 @@ public static async Task<string> PostAndReadResponseContentAsync(
     public static async Task<TObject> PostAndReadResponseContentAsync<TObject>(
         this UITestContext context,
         HttpClient client,
-        string requestUri,
-        string json)
-        where TObject : class
-    {
-        using var response = await PostAndGetResponseAsync(client, requestUri, json);
-        var content = await response.Content.ReadAsStringAsync();
-        var parsed = JToken.Parse(content);
+        string json,
+        string requestUri)
```

Comment:
Why is this breaking change necessary?

Reply:
To have the same structure for all extension methods, where the last parameter is the request URI and the previous parameter is either the json or the object that needs to be serialized.